pkg/cli/helm/helm.go

```diff
@@ -20,15 +20,18 @@ import (
 	_ "embed"
 	"errors"
 	"fmt"
+	"net/http"
 	"os"
 	"path/filepath"
 	"strings"
 	"time"
 
+	containerdErrors "github.com/containerd/containerd/remotes/errors"
```

Comment:
We shouldn't use capital letters in import aliases.

Reply:
fixed